utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp

```diff
@@ -853,14 +856,18 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
                 char img_name[img_size];
                 rocalGetImageName(handle, img_name);
                 std::cerr << "\nPrinting image names of batch: " << img_name;
-                int bb_label_count[inputBatchSize];
-                int size = rocalGetBoundingBoxCount(handle, bb_label_count);
-                for (int i = 0; i < (int)inputBatchSize; i++)
-                    std::cerr << "\n Number of box:  " << bb_label_count[i];
-                int bb_labels[size];
-                rocalGetBoundingBoxLabel(handle, bb_labels);
-                float bb_coords[size * 4];
-                rocalGetBoundingBoxCords(handle, bb_coords);
+                RocalTensorList bbox_labels = rocalGetBoundingBoxLabel(handle);
+                RocalTensorList bbox_coords = rocalGetBoundingBoxCords(handle);
+                for(unsigned i = 0; i < bbox_labels->size(); i++) {
+                    int *labels_buffer = reinterpret_cast<int *>(bbox_labels->at(i)->buffer());
+                    float *bbox_buffer = reinterpret_cast<float *>(bbox_coords->at(i)->buffer());
+                    std::cerr << "\n>>>>> BBOX LABELS : ";
```

Comment:
do we really want to print out all bbox values here?

Reply:
Since for classification we display the labels. Planned to add changes in the detection pipeline to print labels and bbox as well please let us know if this can be retained